src/mame/drivers/8080bw.cpp

```diff
@@ -5760,7 +5812,7 @@ GAMEL(1978, cosmicin,    invaders, invaders,  sicv,      mw8080bw_state, empty_i
 GAMEL(1978, galmonst,    invaders, invaders,  sicv,      mw8080bw_state, empty_init,    ROT270, "bootleg (Laguna S.A.)",              "Galaxy Monsters (Laguna S.A. Spanish bootleg of Space Invaders)", MACHINE_SUPPORTS_SAVE, layout_invaders )
 GAMEL(1979, spacecom,    invaders, spacecom,  spacecom,  _8080bw_state,  init_spacecom, ROT270, "bootleg",                            "Space Combat (bootleg of Space Invaders)",                        MACHINE_IMPERFECT_SOUND | MACHINE_SUPPORTS_SAVE, layout_spacecom )
 GAME( 1978, spacerng,    invaders, spacerng,  sitv,      _8080bw_state,  empty_init,    ROT90,  "bootleg (Leisure Time Electronics)", "Space Ranger",                                                    MACHINE_WRONG_COLORS | MACHINE_SUPPORTS_SAVE | MACHINE_IMPERFECT_SOUND ) // Many modifications
-GAMEL(19??, invasion,    invaders, invaders,  invasion,  mw8080bw_state, empty_init,    ROT270, "bootleg (Sidam)",                    "Invasion (Sidam)",                                                MACHINE_SUPPORTS_SAVE, layout_invaders )
+GAMEL(19??, invasion,    invaders, invasion,  invasion,  _8080bw_state,	 empty_init,    ROT270, "bootleg (Sidam)",                    "Invasion (Sidam)",                                                MACHINE_SUPPORTS_SAVE, layout_invaders )
```

Comment:
The tabulation on this line is messed up now – the columns from the driver initialisation function onward are shifted to the right.  Also, you should flag this machine with the `MACHINE_IMPERFECT_SOUND` flag as it’s known to use a unique Sidam sound board.

Reply:
I've added MACHINE_IMPERFECT_SOUND and fixed the wrong tab. I've not yet pushed the code because I need an advice to resolve the "DIP settings" question you arised below...